src/irmin-pack/irmin_pack.mli

```diff
@@ -98,6 +101,9 @@ module Make
       also try to fix the issues. [ppf] is a formatter for progressive
       reporting. [`Fixed] and [`Corrupted] report the number of fixed/corrupted
       entries. *)
+
+  val ro_sync : repo -> unit
```

Comment:
Perhaps a good time to rip out an `EXT` / `Ext` / `Store_extras` module type containing these two functions and include them here with destructive type substitution. We're already duplicating ~10 lines.

Reply:
much cleaner, thanks !